docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -18,43 +18,76 @@ this relationship. The effect of exposure can be measured both in relative
 and absolute terms. The risk ratio, the rate ratio, and the odds ratio are
 relative measures of effect. Risk difference is an absolute measure of effect
 and it is calculated by subtracting the risk of the outcome in exposed group
-from unexposed. [Measure_of_effect]_
-
-.. todo::
-
-   Discuss causal and non-causal associations, link out to general epi page
+from unexposed [Measure_of_effect]_. See the 
+:ref:`measures of excess risk page <excess_risk_measures>` for more 
+information on such measures.
 
 Risk effect in GBD
 ^^^^^^^^^^^^^^^^^^
 
 The measure of risk effect in GBD is usually reported in relative term, namely
 relative risk. It describes the relative relationship between the risk of
 disease Y in the presence of agent X versus in absence of X. Mathematically,
-it's calculated by dividing the incidence rate (or other measure such as the
-excess mortality rate) of the cause in exposed population by the incidence rate
-(or other measure) of the cause in unexposed population for a certain risk factor.
+it's calculated by dividing the rate of some outcome (such as incidence or mortality
+rate) population exposed to a risk by the outcome rate in unexposed population.
 For example, if there are A incident cases and B person-years in exposed group;
 C incident cases and D person-years in unexposed group, then the relative risk
-(rate ratio) equals :math:`\frac{AD}{BC}`. Note that there are exceptions as in
-the low birth weight short gestation (LBWSG) risk factor where the relative risk
-is the ratio of all-cause mortality rate (ACMR) rather than incidence rate we
-mentioned above. Therefore, we'd better check with GBD modeller what relative
-risk they refer to before we model any risk-outcome pair in vivarium.
+(rate ratio) equals :math:`\frac{AD}{BC}`. The "unexposed" or "reference" group
+in the GBD-estimated relative risks is always the TMREL for that risk. For GBD 
+risk factors with continuous risk exposures, the GBD-estimated relative risks
+represent the relative risk associated with a defined unit increase in risk exposure
+above the TMREL. This unit increase may be more than a single unit change in risk
+exposure, so it is important to clarify this with the GBD modeler.
 
 .. todo::
 
-   Discuss the star rating system for quality of evidence
-
-   Link to mediation
+    Determine if there is a data source that documents these units
+
+The GBD-estimated relative risks are adjusted for confounding and therefore
```

Comment:
Not that we need to break down how to mathematically adjust for confounding -- we should just note above that that is how it is calculated *in the absence of confounders*.

Reply:
Good point, I changed this sentence to reflect that and also added that GBD does an assessment of risk/outcome pairs to make sure they satisfy causal criteria (like absence of reverse causation). I didn't want to get too in the weeds here because I'm thinking it's covered in the burden of proof papers that are linked and the existing vivarium research causal inference page.